prow/cmd/sinker/main.go

```diff
@@ -46,7 +46,6 @@ import (
 	"k8s.io/test-infra/prow/logrusutil"
 	"k8s.io/test-infra/prow/metrics"
 	"k8s.io/test-infra/prow/pjutil"
-	"k8s.io/test-infra/prow/version"
```

Comment:
The `version` package still needs to be imported in order for the `init` function it contains to run. We should force an import for side effects like this: 
```go
_ "k8s.io/test-infra/prow/version"
```

Reply:
good catch, totally forgot about this

Reply:
updated in the package that currently exposing prow_version metrics, could expand this to other packages as well. (Ran `depth -explain k8s.io/test-infra/prow/version k8s.io/test-infra/prow/cmd/hook` took unexpected amount of time :( )